module/VuFindConsole/src/VuFindConsole/Command/Install/InstallCommand.php

```diff
@@ -894,37 +906,41 @@ protected function failWithError(
      */
     protected function displaySuccessMessage(OutputInterface $output)
     {
-        $output->writeln(
-            "Apache configuration written to {$this->overrideDir}/httpd-vufind.conf."
-            . "\n\nYou now need to load this configuration into Apache."
-        );
-        $this->getApacheLocation($output);
-        if (!empty($this->host)) {
+        if ($this->showApacheHelp) {
             $output->writeln(
-                'Since you are using a host-based multisite configuration, you will '
-                . "also \nneed to do some virtual host configuration. See\n"
-                . "     http://httpd.apache.org/docs/2.4/vhosts/\n"
+                "Apache configuration written to {$this->overrideDir}/httpd-vufind.conf."
+                . "\n\nYou now need to load this configuration into Apache."
             );
-        }
-        if ('/' == $this->basePath) {
+            $this->getApacheLocation($output);
+            if (!empty($this->host)) {
+                $output->writeln(
+                    'Since you are using a host-based multisite configuration, you will '
+                    . "also \nneed to do some virtual host configuration. See\n"
+                    . "     http://httpd.apache.org/docs/2.4/vhosts/\n"
+                );
+            }
+            if ('/' == $this->basePath) {
+                $output->writeln(
+                    'Since you are installing VuFind at the root of your domain, you '
+                    . "will also\nneed to edit your Apache configuration to change "
+                    . "DocumentRoot to:\n" . $this->baseDir . "/public\n"
+                );
+            }
             $output->writeln(
-                'Since you are installing VuFind at the root of your domain, you '
-                . "will also\nneed to edit your Apache configuration to change "
-                . "DocumentRoot to:\n" . $this->baseDir . "/public\n"
+                "Once the configuration is linked, restart Apache. You should now be able\n"
+                . "to access VuFind at http://localhost{$this->basePath}"
```

Comment:
We could get this too while we're at it.

```suggestion
                . "to access VuFind® at http://localhost{$this->basePath}"
```

Reply:
I changed them all in the controller while at it.

Reply:
Looks like we need to fix the test as well. I might have time for that later today.

Reply:
Done!